Documentation/driver-api/virtual_bus.rst

```diff
@@ -0,0 +1,162 @@
+===========
+Virtual Bus
+===========
+
+See <linux/virtual_bus.h> for the models for virtbus_device and virtbus_driver.
+
+In some subsystems, the functionality of the core device (PCI/ACPI/other) may
+be too complex for a single device to be managed as a monolithic block.
```

Comment:
Its not just complexity but the core device might have functionality that needs it be exposed / attach to a driver in a different subsystem. Maybe this ok. But I dont want someone to think the virtbus child device and the driver(s) that attach to them all are under one subsystem or class of device.